app/assets/src/components/views/SampleUploadFlow/LocalSampleFileUpload.jsx

```diff
@@ -54,6 +58,12 @@ class LocalSampleFileUpload extends React.Component {
         )} cannot be uploaded. Size must be under 5GB for local uploads. For larger files, please try our CLI.`
     );
 
+  toggleInfo = () => {
```

Comment:
Can you move "Upload Your Input Files" out of the Filepicker so it looks the same as "Path to S3 Bucket"?

<img width="438" alt="screen shot 2019-03-08 at 1 34 45 am" src="https://user-images.githubusercontent.com/837004/54020379-80704880-4142-11e9-9fbb-61de594e22e4.png">

When files are uploaded, I would set the filePickerTitle to "X files uploaded," but make it empty-string when there are no files.

I would also remove "These samples were detected and matched. Remove samples you do not want to upload". It makes sense in the other tab, but it feels unnecessary here.

<img width="813" alt="screen shot 2019-03-08 at 1 37 11 am" src="https://user-images.githubusercontent.com/837004/54020473-b6adc800-4142-11e9-971b-5c6c172b4654.png">

Reply:
👍 

<img width="849" alt="Screen Shot 2019-03-08 at 12 23 32 PM" src="https://user-images.githubusercontent.com/5652739/54053727-10db7700-419d-11e9-9599-67b14b7aa48c.png">
<img width="828" alt="Screen Shot 2019-03-08 at 12 23 24 PM" src="https://user-images.githubusercontent.com/5652739/54053728-10db7700-419d-11e9-9cc1-eb04b4aa0570.png">

Reply:
Could we make the height of the filepicker smaller when there's no title?

Reply:
👍 

![Screen Shot 2019-03-08 at 1 01 35 PM](https://user-images.githubusercontent.com/5652739/54055572-5ea6ae00-41a2-11e9-9fd3-79012088c3ff.png)